services-worker/src/main/java/edu/unc/lib/dl/cdr/services/techmd/TechnicalMetadataEnhancement.java

```diff
@@ -257,10 +258,10 @@ public Element call() throws EnhancementException {
 					(System.currentTimeMillis() - startJob));
 		} catch (FileSystemException e) {
 			throw new EnhancementException(e, Severity.FATAL);
-		} catch (NotFoundException e) {
-			throw new EnhancementException(e, Severity.UNRECOVERABLE);
-		} catch (FedoraException e) {
+		} catch (ObjectExistsException | OptimisticLockException e) {
 			throw new EnhancementException(e, Severity.RECOVERABLE);
+		} catch (FedoraException e) {
```

Comment:
should this be limited to catching a FedoraException? is it impossible that an exception other than FileSystemException, ObjectExistsException, or OptimisticLockException could get thrown by this method?

Reply:
Only FedoraException is explicitly thrown by the code, but there do seem to be some uncaught spring exceptions that come out of there and the other enhancements are catching Exception, so I'll bring it in line with the others.